experimental/segmenter/Cargo.toml

```
@@ -52,10 +52,12 @@ serde_json = "1.0"
 icu = { path = "../../components/icu", features = ["icu_segmenter"]}
 
 [features]
+default = ["lstm", "auto"]
 std = ["icu_collections/std", "icu_locid/std", "icu_provider/std"]
 serde = ["dep:serde", "zerovec/serde", "icu_collections/serde", "icu_provider/serde"]
 datagen = ["serde", "dep:databake", "zerovec/databake", "icu_collections/databake"]
 lstm = ["ndarray", "num-traits"]
```

```suggestion
lstm = ["dep:ndarray", "dep:num-traits"]
```
to not generate implicit features for the dependencies.

---

Nice catch!